tests/test_interface.py

```diff
@@ -396,24 +396,63 @@ def test_interface_query_payments_error(self, mock_db):
 
     #Testing delete/remove payments
     @mock.patch.object(app_db, 'session')
-    def test_interface_delete_payment_with_valid_id(self, mock_db):
-        #payment_to_be_deleted = mock_db.query(Payment).get(1)
-        result = self.ps.remove_payment(payment_id=1)
-        payment_after_deletion = mock_db.query(Payment).get(1)
-        self.assertTrue(payment_after_deletion.is_removed)
-        #TODO - next sprint : do get, check count, should be one less
-    
+    def test_interface_delete_payment_with_valid_id_mock(self, mock_db):
+        payment_id = 1
+
+        cc_payment = Payment()
+        cc_payment.deserialize(CC_RETURN)
+        cc_payment.id = 1
+
```

Comment:
setting the cc_payment.id to 1 doesn't have any effect on the delete operation since it acts on the parameter passed to remove_payments. You could set cc_payment.id = 2139230 and all tests would still pass.

so i'd just scrap that line. otherwise looks good to me!